policy-build-rust/action.yaml

```diff
@@ -35,3 +35,51 @@ runs:
       shell: bash
       run: |
         kwctl annotate -m ${{ inputs.metadata-file }} -o annotated-policy.wasm ${{ inputs.input-wasm }}
+    -
+      name: Generate the SBOM files
+      shell: bash
+      run: |
+        curl -Lo $RUNNER_TEMP/sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
+        chmod +x $RUNNER_TEMP/sbom-tool
+        $RUNNER_TEMP/sbom-tool generate -D true \
+          -V Verbose \
+          -b ./target/wasm32-wasi/release/ \
+          -bc . \
+          -m . \
+          -nsb https://kubewarden.io \
+          -nsu kubewarden-policy \
+          -pn pod-pribileged-policy \
+          -pv test
```

Comment:
I suppose the package name shouldn't be hardcoded, or the package version set to `test`

Reply:
Ops...